dd-java-agent/instrumentation/jersey/src/main/java/datadog/trace/instrumentation/jersey/AbstractStringReaderInstrumentation.java

```diff
@@ -0,0 +1,30 @@
+package datadog.trace.instrumentation.jersey3;
+
+import static datadog.trace.agent.tooling.bytebuddy.matcher.NameMatchers.named;
+import static net.bytebuddy.matcher.ElementMatchers.isPublic;
+
+import com.google.auto.service.AutoService;
+import datadog.trace.agent.tooling.Instrumenter;
+
+@AutoService(Instrumenter.class)
+public class AbstractStringReaderInstrumentation extends Instrumenter.Tracing
```

Comment:
```suggestion
public class AbstractStringReaderInstrumentation extends Instrumenter.Iast
```

So that they are activated only with IAST.